packages/rocketchat-videobridge/client/public/external_api.js

```diff
@@ -0,0 +1,377 @@
+        }
+        this.sendMessage(message);
+    };
+
+    /**
+     * Adds event listeners to Meet Jitsi. Currently we support the following
+     * events:
+     * incomingMessage - receives event notifications about incoming
+     * messages. The listener will receive object with the following structure:
+     * {{
+     *  "from": from,//JID of the user that sent the message
+     *  "nick": nick,//the nickname of the user that sent the message
+     *  "message": txt//the text of the message
+     * }}
+     * outgoingMessage - receives event notifications about outgoing
+     * messages. The listener will receive object with the following structure:
+     * {{
+     *  "message": txt//the text of the message
+     * }}
+     * displayNameChanged - receives event notifications about display name
+     * change. The listener will receive object with the following structure:
+     * {{
+     * jid: jid,//the JID of the participant that changed his display name
+     * displayname: displayName //the new display name
+     * }}
+     * participantJoined - receives event notifications about new participant.
+     * The listener will receive object with the following structure:
+     * {{
+     * jid: jid //the jid of the participant
+     * }}
+     * participantLeft - receives event notifications about participant the that
+     * left the room.
+     * The listener will receive object with the following structure:
+     * {{
+     * jid: jid //the jid of the participant
+     * }}
+     * @param event the name of the event
+     * @param listener the listener
+     */
+    JitsiMeetExternalAPI.prototype.addEventListener
+        = function (event, listener) {
+
+        var message = {type: "event", action: "add", events: [event]};
+        this.eventHandlers[event] = listener;
+        this.sendMessage(message);
+    };
+
+    /**
+     * Removes event listener.
+     * @param event the name of the event.
+     */
+    JitsiMeetExternalAPI.prototype.removeEventListener
+        = function (event) {
+        if(!this.eventHandlers[event])
+        {
+            console.error("The event " + event + " is not registered.");
+            return;
+        }
+        var message = {type: "event", action: "remove", events: [event]};
+        delete this.eventHandlers[event];
+        this.sendMessage(message);
+    };
+
+    /**
+     * Removes event listeners.
+     * @param events array with the names of the events.
+     */
+    JitsiMeetExternalAPI.prototype.removeEventListeners
+        = function (events) {
+        var eventsArray = [];
+        for(var i = 0; i < events.length; i++)
+        {
+            var event = events[i];
+            if(!this.eventHandlers[event])
+            {
+                console.error("The event " + event + " is not registered.");
+                continue;
+            }
+            delete this.eventHandlers[event];
+            eventsArray.push(event);
+        }
+
+        if(eventsArray.length > 0)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Keyword "if" must be followed by whitespace.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2737452894/issues/source?bid=3199634&fileBranchId=3342079#l282)